### ISSUE TITLE
[Improvement]: Improve pull request CI build for handwritten connectors

### ISSUE THREAD

**Issue by Chuhaa** [ballerina-platform/ballerina-extended-library#108]
### Connector Name

other (None Listed Above)

### Suggested improvement

Currently pull request CI build is failing in most of the handwritten connectors due to inaccessibility of ballerina secrets. Therefore we are able to validate the build once we merge the PR and then by looking ballerina repository CI build. 

If we skip the test cases and only do the build CI in the pull request CI, we would be able to identify build failure before merging a PR.

**Comment by LakshanSS** [comment]
When we send a PR from our cloned repo, we dont get access to the github sectrets. Thats why the test cases fail.

If we create a branch on the original repo and send a PR, we can access the github secrets. So test cases will pass.

**Comment by Chuhaa** [comment]
We could skip tests in the PR CI and check only build by using following steps.
1. If build has java dependency, CI token should be given as 
```
env:
          packageUser: ${{ github.actor }}
          packagePAT: ${{ secrets.GITHUB_TOKEN }}
```
2. Skip test cases in the pull request request

Through above approach, we might be able to verify the build before merging cloned repo PR.

Refer https://github.com/ballerina-platform/module-ballerinax-sfdc/blob/master/.github/workflows/pull-request.yml

**Comment by NipunaRanasinghe** [comment]
These improvements are added as a part of https://github.com/ballerina-platform/ballerina-extended-library/issues/539 effort. Hence closing the issue.
